pyface/ui/qt4/tasks/split_editor_area_pane.py

```diff
@@ -939,18 +939,18 @@ def get_pixmap(self):
         painter.drawPrimitive(QtGui.QStyle.PE_FrameTabBarBase, optTabBase)
 
         # region of active tab
-        if QtGui.qt_api=='pyqt5':
-            pixmap1 = tabBar.grab()
-        else: # pyqt4 (grabWidget was removed in pyqt5)
+        if is_qt4:  # grab wasn't introduced until Qt5
             pixmap1 = QtGui.QPixmap.grabWidget(tabBar, tab_rect)
+        else:
+            pixmap1 = tabBar.grab(tab_rect)
 
-        painter.drawPixmap(0,0,pixmap1) #tab_rect.topLeft(), pixmap1)
+        painter.drawPixmap(0, 0, pixmap1)  # tab_rect.topLeft(), pixmap1)
```

Comment:
What is the significance of the commented out code? Can we throw it out? (I understand that this might have been a drive-by change).

Reply:
Yes, I suspect we should throw it out, unless anyone remembers why it was changed in the first place.